src/queries/get-workflow-templates.js

```diff
@@ -0,0 +1,40 @@
+const polarityRequest = require('../polarity-request');
+const { ApiRequestError } = require('../errors');
+const { getLogger } = require('../logger');
+const SUCCESS_CODES = [200];
+
+async function getWorkflowTemplates(options) {
+  const Logger = getLogger();
+
+  const requestOptions = {
+    uri: `${options.url}/v3/workflowTemplates`,
+    qs: {
+      tql: `active EQ true`
+    },
+    method: 'GET'
+  };
+
+  Logger.trace({ requestOptions }, 'Workflow Templates Request Options');
+
+  const apiResponse = await polarityRequest.request(requestOptions, options);
+
+  if (
+    !SUCCESS_CODES.includes(apiResponse.statusCode) ||
+    (apiResponse.body && apiResponse.body.status && apiResponse.body.status !== 'Success')
+  ) {
+    throw new ApiRequestError(
+      `Unexpected status code ${apiResponse.statusCode} received when fetching Workflow Templates via the ThreatConnect API`,
+      {
+        statusCode: apiResponse.statusCode,
+        requestOptions: apiResponse.requestOptions,
+        responseBody: apiResponse.body
+      }
+    );
+  }
+
+  return apiResponse;
```

Comment:
Most of our other endpoints directly return the data from the response which is what I would do just to stay consistent.  This will also make it simpler in the component so you don't have to do:

```
this.set(`${path}.workflowTemplates`, response.data.body.data);
```

Instead you could get away with:

```
this.set(`${path}.workflowTemplates`, response.worksflows);
```

So I would probably return the `data` property here to be consistent:

```
return apiResponse.data
```